docs/get-started/architecture/introduction.rst

```diff
@@ -147,12 +147,12 @@ Determined Notebooks have the following benefits:
 
 .. note::
 
-   -  Once a Notebook is terminated, it is not possible to restore the files that are not stored in
+   -  Once a notebook is terminated, it is not possible to restore the files that are not stored in
```

Comment:
where is the style guide for capitalization of Notebook vs notebook?

This feels like a case where it would be capitalized to me, but I don't remember where the style guide is, ha